serialization/physics.proto

```diff
@@ -1,4 +1,4 @@
-syntax = "proto2";
+﻿syntax = "proto2";
```

Comment:
I think when run from visual studio (as opposed to from msbuild itself called from jenkins's shell) the BOM in the protos causes issues. Set encoding to UTF-8 without BOM, see the comment on #924.